src/shared/utilities/childProcess.ts

```diff
@@ -106,7 +106,7 @@ export class ChildProcess {
             throw new Error('process already started')
         }
 
-        this.log.info(`Running: ${this.toString()}`)
+        this.log.info(`Running: ${this.toString(this.options.logging === 'noparams')}`)
```

Comment:
Potential follow-up:
Don't display params unless in debug/verbose mode (`noparams` overrides all)

Reply:
Maybe. Though I wonder what cases we would want to log a command at INFO level but not show the params. If it's sensitive info, we pretty much will never log it.

Reply:
Yeah I guess that's true, though the exact parameters for every command just seems noisy
Could we also pattern match for potentially sensitive things? Maybe even trim excessively long strings.

Reply:
>  though the exact parameters for every command just seems noisy

It's noisy but crucial for visibility into what's actually going on, for things like SAM CLI where we (and customers) need to reproduce the steps at the CLI. 

If it's too noisy at a particular log-level then we should just increment the log-level.

> Could we also pattern match for potentially sensitive things? Maybe even trim excessively long strings.

Definitely worth exploring at some point.